bigtable-hbase-2.x-parent/bigtable-hbase-2.x/src/main/java/com/google/cloud/bigtable/hbase2_x/BigtableAsyncAdmin.java

```diff
@@ -601,4 +640,67 @@ public static BigtableAsyncAdmin createInstance(CommonConnection connection) thr
       throw new IOException(e);
     }
   }
+
+  private static Class<? extends RegionMetrics> regionMetricsClass = null;
+
+  private static synchronized Class<? extends RegionMetrics> getRegionMetricsSubclass()
+      throws NoSuchMethodException {
+    if (regionMetricsClass == null) {
+      regionMetricsClass =
+          new ByteBuddy()
+              .subclass(BasicRegionMetrics.class)
+              .name(BasicRegionMetrics.class.getName() + "Impl")
```

Comment:
why + "Impl"?

Reply:
To avoid the autogenerated ByteBuddyName